text/0000-modifiers.md

```diff
@@ -0,0 +1,394 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Element Modifiers
+
+## Summary
+
+This RFC introduces the concept of user defined element modifiers. Unlike a component, there is no template/layout for an element modifier. Unlike a helper, an element modifier does not return a value.
+
+Below is an example of the element modifier syntax:
+
+```hbs
+<button {{add-event-listener 'click' (action 'save')}}>Save</button>
```

Comment:
First, I totally like this RFC! Just this example was a bit confusing for me when first reading this, as it does not show any real value added here, IMHO. To my understanding, this would be equivalent to:
```hbs
<button {{action 'save'}}>Save</button>
```
At least from the user's perspective (ignoring internals like the action modifiers use of `data-ember-action` and the resulting event delegation).

Maybe another example thats show something *not* easily possible today would be nice, like
```hbs
<button {{effect 'fade-in'}}>Save</button>
```